### DISCUSSION TITLE
额度消费问题

### DISCUSSION THREAD

**Discussion by ipf37200** [BurnHair/docs#168] [技术支持]
感谢提供此服务, 虽然本人白嫖只是有需要时才拿来当作翻译软件.
最近闲着没事看了一下 消费纪录 因为我主要是用4o, 发现除了4o 还有一个3.5会伴随出现

我每次用都会开一个新对话然后选4o, 这个3.5会是因为我之前的对话中有用过3.5的关系吗?
![image](https://github.com/user-attachments/assets/61f5e74a-0b1d-4384-8206-5cb2cc7eef35)

**Comment by BennyThink** [comment]
https://github.com/BurnHair/docs/blob/master/FAQ.md#5-为什么日志里显示的模型来回变我明明只使用4
